src/pages/Search/style.css

```diff
@@ -0,0 +1,11 @@
+main {
```

Comment:
Use a class to be more specific, otherwise this will be applied to every <main> across the project